spec/room_spec.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'spec_helper'
+
+describe "Room Class" do
+  before do
+    room_num = 1
+    @room = Hotel::Room.new(room_num)
+  end
+  describe "Initializer" do
+    it "is an instance of Room" do
+      expect(@room).must_be_kind_of Hotel::Room
+      expect(@room.room_reservations).must_be :empty?
+    end
+
+    it "must keep track of room number" do
+      expect(@room).must_respond_to :room_num
+      expect(@room.room_num).must_be_kind_of Integer
+    end
+
+    it "must keep track of reservations" do
+      expect(@room).must_respond_to :room_reservations
+      expect(@room.room_reservations).must_be_kind_of Array
+      expect(@room.room_reservations).must_be :empty?
+
+
+      @room.room_reservations.each do |room_reservation|
```

Comment:
You just asserted that the list of reservations is empty, so it doesn't make any sense to iterate through it. Perhaps you could add to the list first? Though that probably ought to be a separate test.